stripe/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+ - **FEAT**: first melos version.
```

Comment:
I do not think this is a user facing feature maybe call it something like: "bug fixes"